core/jazz_acl/components/validation.js

```diff
@@ -16,23 +16,47 @@
 
 const errorHandlerModule = require("./error-handler.js")();
 
-async function validateInput(userInput) {
+function validateBasicInput(userInput) {
 
 		if (!userInput || !userInput.method) {
 			throw (errorHandlerModule.throwInputValidationError("Invalid or missing arguments"));
```

Comment:
Please separate these out. 

- method is usually set by API Gateway. If it's missing, it's more of a system integration error (InternalServerError).
- if `userInput` is missing, it's `InputValidationError`